### PR TITLE
visit_ImportFrom fix and NamedExpr not implemented

### PR DESCRIPTION
**Pull request by jayvdb**
AttributeError occurs because node.module is None
when `from . import ...`

### CODE REVIEW

File: py2many/clike.py [hunk 1]
```diff
@@ -527,6 +527,10 @@ def visit_unsupported_body(self, node, name, body):
                 f"{name} unimplemented on line {node.lineno}:{node.col_offset}"
             )
 
+    def visit_NamedExpr(self, node):
+        target = self.visit(node.target)
```
Comment (adsharma) [inline py2many/clike.py:531]: visit node.value as well?

**Review by adsharma — COMMENTED** [review]
https://www.python.org/dev/peps/pep-0572/

**Review by adsharma — COMMENTED** [review]
typo on Pythoon

File: py2many/clike.py (hunk 1 above)
Comment (jayvdb) [reply py2many/clike.py:531]: I've passed that as the `body` parameter to `visit_unsupported_body`, similar to the other uses.
I expect we'll enhance `visit_unsupported_body` to do that visiting.

**Pull request merged**